inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -51,16 +51,13 @@
   <a class="btn btn-default dropdown-toggle dropdown-cite cite-btn" type="button" id="dropdownMenu{{record['control_number']}}" data-recid="{{record['control_number']}}"  data-toggle="modal" data-target="#citeModal{{record['control_number']}}">
     <i class="fa fa-quote-right"></i> Cite this record
   </a>
-  {% if record.get('url') %}
-    {% set exists = [] %}
-    {% for url in record.get('url') %}
-      {% if url.get('url').endswith(".pdf") %}
-        {% if not exists %}
-          {% do exists.append(1) %}  
-            <a class="btn custom-btn btn-warning pdf-btn" href="{{ url.get('url') }}" target="_blank" role="button" title="View PDF"><i class="fa fa-eye"></i> View PDF</a>
-        {% endif %}
-      {% endif %}
-    {% endfor %}
+  {% if record.get('arxiv_eprints') %}
+    {% if record.get('arxiv_eprints') | is_list() %}
+      {% set filtered_arxiv = record.get('arxiv_eprints') %}
+      {% for i in filtered_arxiv %}
+        <a class="btn custom-btn btn-warning pdf-btn" href="http://arxiv.org/pdf/{{ i.get('value') }}" role="button" title="View PDF"><i class="fa fa-eye"></i> View PDF</a>
```

Comment:
Looking at the code that we use nowadays to link to arXiv records, I assume this link will not always work.

You can create a filter that does the same logic as today:

https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_arxiv_link.py#L32-L47